src/frontend/src/shared/Button/SharedButton.js

```diff
@@ -1,14 +1,12 @@
-import Style from './styles'
+import styles from './styles'
 import Button from '@material-ui/core/Button'
 
-function SharedButton ({title, color = 'primary', variant = 'contained', disabled = false}) {
-  const classes = Style()
+function SharedButton ({ title, color = 'primary', variant = 'contained', disabled = false, size = 'small' }) {
```

Comment:
We should pass OnClick function as an argument to SharedButton component.
Then it will be able to pass down this function to it's child: Button component.
Because there is one place in AddNewPost.js (approximately line 156) where Button was replaced with SharedButton.
<SharedButton title="Post" disabled={btnIsDisabled} onClick={onPostSubmitHandler}/>

In general, that is fine. But SharedButton is not waiting for onClick argument at the moment. And therefore it can not pass it further to it's child - Button. That's why in this branch our functionality of sending new Post doesn't work.
This should be fixed.